### ISSUE TITLE
The program closes immediately after launch

### ISSUE THREAD

**Issue by Niotferdi** [master-of-zen/Av1an#784]
I downloaded the av1an binary and put the FFmpeg, VapourSynth and SvtAv1EncApp binaries in the same folder. I type av1an.exe while in its folder, but I don't get any output, the program quits after a second.
What am I doing wrong?

**Comment by michaelbub** [comment]
Are you in PowerShell? Just had the same problem. Even ".\av1an.exe --help" did not give any output. So I ran it from a regular old shell and little windows would pop up showing me info about all the DLLs that could not be found.

**Comment by menditsa** [comment]
You can run from within powershell.
Your problem indicates a problem with paths or binaries.
FFMPEG needs shared build not static.

**Comment by michaelbub** [comment]
> You can run from within powershell.
> Your problem indicates a problem with paths or binaries.
> FFMPEG needs shared build not static.

Yes, but when a DLL is missing and you run via PowerShell there is no pop-up informing you which DLL is missing (Windows 11). So if you indeed have a problem with DLLs you will never learn what it is. Only via regular shell do the pop-ups show. No idea what the problem is with these pop-ups, but that is what I experienced.

**Comment by foxmsft** [comment]
The program insta-crashes here in `vsscript_createScript` with access violation. I'm not sure it's in a state where it can reproducibly build on Windows right now, I'm using the `HEAD` from here, plus AviSynth R58 (since it's linked in the compilation document), plus an ffmpeg 6.1.
